README.md

```diff
@@ -68,6 +71,28 @@ Pion DTLS can connect to itself and OpenSSL.
   openssl s_client -dtls1_2 -connect 127.0.0.1:4444 -debug -cert cert.pem -key key.pem
 ```
 
+### Using with PSK
+Pion DTLS also comes with examples that do key exchange via PSK
+
+
+#### Pion DTLS
+```sh
+go run examples/listen-psk/main.go
+```
+
+```sh
+go run examples/dial-psk/main.go
+```
+
+#### OpenSSL
+```
+  // Use with examples/dial-psk/main.go
+  openssl s_server -dtls1_2 -accept 4444 -nocert -psk abc123 -cipher PSK-AES128-CCM8
+
+  // Use with examples/listen-psk/main.go
+  openssl s_client -dtls1_2 -connect 127.0.0.1:4444 -psk abc123 -cipher PSK-AES128-CCM8
+```
```

Comment:
I think these two examples need to be swapped? dial-psk should be `s_client`, not `s_server`?